DQMOffline/Trigger/plugins/MssmHbbMonitor.cc

```diff
@@ -0,0 +1,203 @@
+  muonPtbins_             = getHistoPSet(iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<edm::ParameterSet>("muonPt"));
+  muonEtabins_            = getHistoPSet(iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<edm::ParameterSet>("muonEta"));
+  muonPhibins_            = getHistoPSet(iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<edm::ParameterSet>("muonPhi"));
+  
+}
+
+MssmHbbMonitor::~MssmHbbMonitor()
+{
+
+}
+
+Binning MssmHbbMonitor::getHistoPSet(edm::ParameterSet pset)
+{
+   return Binning
+   {
+      pset.getParameter<int32_t>("nbins"),
+      pset.getParameter<double>("xmin"),
+      pset.getParameter<double>("xmax"),
+   };
+}
+
+
+
+void MssmHbbMonitor::bookHistograms(DQMStore::IBooker     & ibooker,
+				 edm::Run const        & iRun,
+				 edm::EventSetup const & iSetup) 
+{  
+   
+  std::string currentFolder = folderName_ ;
+  ibooker.setCurrentFolder(currentFolder.c_str());
+  
+  pt_jet1_  = ibooker.book1D("pt_jet1","pt_jet1",jetPtbins_.nbins,jetPtbins_.xmin,jetPtbins_.xmax);
+  pt_jet2_  = ibooker.book1D("pt_jet2","pt_jet2",jetPtbins_.nbins,jetPtbins_.xmin,jetPtbins_.xmax);
+  eta_jet1_ = ibooker.book1D("eta_jet1","eta_jet1",jetEtabins_.nbins,jetEtabins_.xmin,jetEtabins_.xmax);
+  eta_jet2_ = ibooker.book1D("eta_jet2","eta_jet2",jetEtabins_.nbins,jetEtabins_.xmin,jetEtabins_.xmax);
+  phi_jet1_ = ibooker.book1D("phi_jet1","phi_jet1",jetPhibins_.nbins,jetPhibins_.xmin,jetPhibins_.xmax);
+  phi_jet2_ = ibooker.book1D("phi_jet2","phi_jet2",jetPhibins_.nbins,jetPhibins_.xmin,jetPhibins_.xmax);
+  
+  eta_phi_jet1_ = ibooker.book2D("eta_phi_jet1","eta_phi_jet1",jetEtabins_.nbins,jetEtabins_.xmin,jetEtabins_.xmax,jetPhibins_.nbins,jetPhibins_.xmin,jetPhibins_.xmax);
+  eta_phi_jet2_ = ibooker.book2D("eta_phi_jet2","eta_phi_jet2",jetEtabins_.nbins,jetEtabins_.xmin,jetEtabins_.xmax,jetPhibins_.nbins,jetPhibins_.xmin,jetPhibins_.xmax);
+  
+  deta_jet12_ = ibooker.book1D("deta_jet12","deta_jet12",jetEtabins_.nbins,jetEtabins_.xmin,jetEtabins_.xmax);
+  dphi_jet12_ = ibooker.book1D("dphi_jet12","dphi_jet12",jetPhibins_.nbins,jetPhibins_.xmin,jetPhibins_.xmax);
+  dr_jet12_   = ibooker.book1D("dr_jet12","dr_jet12",jetDRbins_.nbins,jetDRbins_.xmin,jetDRbins_.xmax);
+
+  discr_offline_btag_jet1_ = ibooker.book1D("discr_offline_btag_jet1","discr_offline_btag_jet1",jetBtagbins_.nbins,jetBtagbins_.xmin,jetBtagbins_.xmax);
+  discr_offline_btag_jet2_ = ibooker.book1D("discr_offline_btag_jet2","discr_offline_btag_jet2",jetBtagbins_.nbins,jetBtagbins_.xmin,jetBtagbins_.xmax);
+
+  pt_muon_  = ibooker.book1D("pt_muon","pt_muon",muonPtbins_.nbins,muonPtbins_.xmin,muonPtbins_.xmax);
+  eta_muon_ = ibooker.book1D("eta_muon","eta_muon",muonEtabins_.nbins,muonEtabins_.xmin,muonEtabins_.xmax);
+  phi_muon_ = ibooker.book1D("phi_muon","phi_muon",muonPhibins_.nbins,muonPhibins_.xmin,muonPhibins_.xmax);
+    
+  // Initialize the GenericTriggerEventFlag
```

Comment:
uhmm, I still do not see the GenericTriggerEventFlag
in this way runs w/ some LSs w/ tracker HV off will pollute your distribution ....